app/main.py

```diff
@@ -1,3 +1,51 @@
 class Distance:
-    # Write your code here
-    pass
+    def __init__(self, km: int) -> None:
+        self.km = km
+
+    def __str__(self) -> str:
+        return f"Distance: {self.km} kilometers."
+
+    def __repr__(self) -> str:
+        return f"Distance(km={self.km})"
+
+    def __add__(self, other: None) -> None:
+        if isinstance(other, Distance):
+            return Distance(km=self.km + other.km)
+        elif isinstance(other, (int, float)):
+            return Distance(km=self.km + other)
+        else:
+            raise TypeError(f"unsupported operand type(s) for +: "
+                            f"Distance and {type(other)}")
+
+    def __iadd__(self, other: None) -> None:
+        if isinstance(other, Distance):
+            self.km += other.km
+        elif isinstance(other, (int, float)):
+            self.km += other
+        else:
+            raise TypeError(f"unsupported operand type(s) for +: "
+                            f"Distance and {type(other)}")
+        return self
+
+    def __mul__(self, koef: int) -> None:
```

Comment:
The `__mul__` method should have a return type of `Distance` instead of `None` in its type annotation.